pox-locking/src/events.rs

```diff
@@ -362,7 +364,8 @@ pub fn synthesize_pox_2_or_3_event_info(
         | "delegate-stack-extend"
         | "stack-increase"
         | "delegate-stack-increase"
-        | "delegate-stx" => Some(create_event_info_stack_or_delegate_code(
+        | "delegate-stx"
+        | "revoke-delegate-stx" => Some(create_event_info_stack_or_delegate_code(
```

Comment:
@friedger I'm looking into ingesting this new event, and it seems to have issues. It looks like this event is missing code in the above `create_event_info_data_code` function. Can you fix that?

And can you please add tests for this event similar to how the others are tested? Here's a PR showing how tests for the `delegate-stx` event were implemented: https://github.com/stacks-network/stacks-core/pull/3503/files

Reply:
@zone117x I have added the missing code. However, the test does not pass. The event is not created. Do you have any hints what is missing?

Reply:
It might be related to https://github.com/stacks-network/stacks-core/issues/4143

Reply:
Moved to https://github.com/stacks-network/stacks-core/pull/4157